epregressions/tk_window.py

```diff
@@ -85,9 +85,10 @@ def __init__(self):
         self.root = Tk(className='eplus_regression_runner')
         Frame.__init__(self, self.root)
 
-        # add the taskbar icon
-        img = PhotoImage(file=os.path.join(os.path.dirname(os.path.abspath(__file__)), 'ep.png'))
-        self.root.iconphoto(False, img)
+        # add the taskbar icon, but its having issues reading the png on Mac, not sure.
+        if system() != 'Darwin':
+            img = PhotoImage(file=os.path.join(os.path.dirname(os.path.abspath(__file__)), 'ep.png'))
+            self.root.iconphoto(False, img)
```

Comment:
I super don't know why Mac started having trouble with the icon, but in any case, on Darwin we are not setting a program icon.